### PR TITLE
New Tip

### PR DESCRIPTION
**Pull request by redmangospros**

### CODE REVIEW

**Comment by redmangospros** [comment]
Ping @vutondesign 
Ping @redmango22

File: index.html [hunk 1]
```diff
@@ -42,6 +42,7 @@ <h4> Hosting </h4>
      <li>- Choose whats best for you, hosting your site locally or using services like GitHub Pages or Forge. </li>
      <li>- Make the most of the services your hosting provider gives you. (Like a business email address or image storage.)</li>
      <li>- Check how much downtime your site host has, if it's a lot then consider moving.</li>
+        <li> Chose A Hosting service, what is safe and many people use </li>
```
Comment (vutondesign) [inline index.html]: The indentation seems to be a bit off, along with the spelling and you seems to be missing a `-` after the `<li>

**Comment by redmangospros** [comment]
Have you seen this yet?

**Comment by vutondesign** [comment]
@redmangospros Yes, please take a look at my review above.

File: index.html (hunk 1 above)
Comment (redmangospros) [reply index.html]: What do you mean?

Comment (redmangospros) [reply index.html]: Oh okay, i got it

File: index.html [hunk 2]
```diff
@@ -42,7 +42,7 @@ <h4> Hosting </h4>
      <li>- Choose whats best for you, hosting your site locally or using services like GitHub Pages or Forge. </li>
      <li>- Make the most of the services your hosting provider gives you. (Like a business email address or image storage.)</li>
      <li>- Check how much downtime your site host has, if it's a lot then consider moving.</li>
-        <li> Chose A Hosting service, what is safe and many people use </li>
+     <li>- Chose A Hosting service, what is safe and many people use </li>
```
Comment (vutondesign) [inline index.html]: Should be - 
Ch**o**ose **a** **h**osting service **that** is safe and **trusted by many people**.

Comment (redmangospros) [reply index.html]: Okay!

**Comment by redmangospros** [comment]
Can you do it now?

**Comment by vutondesign** [comment]
I need to run your enhancement locally first.

**Comment by redmangospros** [comment]
what do you mean?

**Comment by vutondesign** [comment]
I need to copy your changes and see what it looks like on the site.

**Comment by redmangospros** [comment]
Okay.

**Comment by redmangospros** [comment]
This it how it looks
screenshot 2017-12-23 at 11 21 11 am

**Comment by vutondesign** [comment]
Done ✅

**Comment by redmangospros** [comment]
Your Welcome.

**Comment by vutondesign** [comment]
Oh, and congrats on your first PR here! 🎈

**Comment by redmangospros** [comment]
Thanks! I have aleast 3 now.

**Pull request merged**